app/models/artist.rb

```diff
@@ -16,28 +16,20 @@ class Artist < ActiveRecord::Base
 
   validates_confirmation_of :password, on: :create
 
+  scope :activated, -> { where(activated: true) }
+  scope :funded, -> do
+    joins(:grant_submissions)
+      .where("grant_submissions.funding_decision == 't'")
```

Comment:
Is there any way to make this something like .where(grant_submissions.funding_decision: true) ?

Reply:
Yep, you can use `.where("grant_submissions.funding_decision": true)`. I just copied the query as is.

Reply:
You can also combine scopes from GrantSubmission.

Reply:
Updated to use the existing `GrantSubmission.funded` scope.